src/pages/projects/forms/NetworkSelector.tsx

```diff
@@ -22,38 +22,89 @@ const NetworkSelector: FC<Props & SelectProps> = ({
   const { data: networks = [] } = useNetworks(project);
 
   const managedNetworks = networks.filter((network) => network.managed);
+  const hasAcls = managedNetworks.some((t) => t.config["security.acls"]);
```

Comment:
```suggestion
```
I think we should always have the ACLs visible in here, thus can remove the const. See https://github.com/canonical/lxd-ui/pull/1513#discussion_r2387678174 for the reasoning.